platform/android/app/src/main/kotlin/coop/polypoly/polypod/PodApi.kt

```diff
@@ -92,10 +92,9 @@ open class PodApi(
         )
     }
 
-    private suspend fun handlePolyOutFetch(args: List<Value>): Value {
+    private suspend fun handlePolyOutFetch(): Value {
```

Comment:
I mean, polyOut.fetch is going away anyway, but for the record, you could have also marked the parameter as unused - seems a bit cleaner to me cause then we'd only have to make changes in this function if we were to actually implement it later:

```
private suspend fun handlePolyOutFetch(@Suppress("UNUSED_PARAMETER") args: List<Value>): Value {
```

Reply:
`suspend` is apparently not necessary either. But let's leave that for later, I guess.